node-problem-detector.yaml

```diff
@@ -26,6 +26,11 @@ spec:
         - name: log
           mountPath: /log
           readOnly: true
+        # Make sure node problem detector is in the same timezone
+        # with the host.
+        - name: localtime
+          mountPath: /etc/localtime
+          readOnly: true
```

Comment:
I read your link and understand what you mean. Thanks for sharing!

IIUC, it means that if `/etc/localtime` inside the container is a symlink (fedora is just using symlink), say `/etc/localtime -> ../usr/share/zoneinfo/UTC`. `../usr/share/zoneinfo/UTC` will be mounted instead.

This should not cause trouble to [golang](https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/time/zoneinfo_unix.go#L59), but it's worth fixing to avoid confusion.